units/6_unit/05b_lesson/lesson.md

```diff
@@ -0,0 +1,78 @@
+# Lesson 6.05: Dictionaries Project -- Buy an Umbrella
+
+## Learning Objectives
+Students will be able to...
+
+* Use dictionaries to create a shopping site where you can browse and filter
+  a catalog of umbrellas before buying one.
+
+## Materials/Preparation
+* [Do Now][0]
```

Comment:
You don't need the `[0]` after this. Just `[Do Now]` on this line, and at the bottom, this:
```
[Do Now]: ./do_now.md
```